src/Apps/CollectorProfile/Routes/Saves/Components/ArtworkListContent.tsx

```diff
@@ -55,11 +56,12 @@ function isContentOutOfView() {
 }
 
 const ArtworkListContent: FC<ArtworkListContentProps> = ({ me, relay }) => {
+  const { t } = useTranslation()
   const { match } = useRouter()
 
   const { jumpTo } = useJump()
   const { artworkListItemHasBeenTouched } = useArtworkListVisibilityContext()
-
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Yeah, I had to add that because I haven't changed that line, so probably a later edit in this file triggered the linter.

With your proposed change, I am getting this.
![image](https://github.com/artsy/force/assets/554507/15e51dae-43af-48ba-b5d2-4d086bc07267)